ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -173,14 +192,14 @@ set_local_repo() {
     version="$(lsb_release -sr)"
     version=${version%%.*}
     if [ "$repo_server" = "artifactory" ] &&
-        [[ $(echo "$COMMIT_MESSAGE" | sed -ne '/^PR-repos: */s/^[^:]*: *//p') = *daos@* ]] ||
-        [[ $(echo "$COMMIT_MESSAGE" |
-             sed -ne "/^PR-repos-$DISTRO: */s/^[^:]*: *//p") = *daos@* ]] ||
-        [ -z "$(echo "$COMMIT_MESSAGE" | sed -ne '/^RPM-test-version: */s/^[^:]*: *//p')" ]; then
-        # Disable the daos repo so that the Jenkins job repo is used for daos packages
+       [ -z "$(rpm_test_version)" ] &&
```

Comment:
Should this be calling `pr_repos()` as well?

Reply:
No.  Keeping the daos repo enabled even if `PR-repos:` points at a daos PR was a misjudgement.  The only time we want the daos repo enabled is if we want to grab an existing build of daos from it, which only happens when `RPM-test-version:` is used or for the `weekly-testing` runs.